src/strategies/base.ts

```diff
@@ -489,13 +489,21 @@ export abstract class BaseStrategy implements Strategy {
       releaseData = pullRequestBody.releaseData[0];
     } else {
       // manifest release with multiple components
-      releaseData = pullRequestBody.releaseData.find(releaseData => {
+      releaseData = pullRequestBody.releaseData.find(datum => {
         return (
-          this.normalizeComponent(releaseData.component) ===
+          this.normalizeComponent(datum.component) ===
```

Comment:
A comment would be useful to me, that mentions that `component` is one of the parsed elements from the pull request body.